client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -64,10 +65,15 @@ class VerificationCodeForm extends Component {
 	};
 
 	onChangeField = ( event ) => {
+		const {
+			name,
+			value = '',
+		} = event.target;
+
 		this.props.formUpdate();
 
 		this.setState( {
-			[ event.target.name ]: event.target.value,
+			[ name ]: typeof value === 'string' ? value.trim() : value,
```

Comment:
In its current state, this component would never set the target value to `undefined` nor `null`. Just my instinctual defensive programming habits kicking in :)

I'll remove this check and merge this PR.